modules/core/src/main/scala/com.snowplowanalytics.snowplow.snowflake/processing/SnowflakeRetrying.scala

```diff
@@ -35,6 +36,10 @@ object SnowflakeRetrying {
 
   /** Is an error associated with setting up Snowflake as a destination */
   private def isSetupError: PartialFunction[Throwable, String] = {
+    case sse: SnowflakeSQLException if sse.isEligibleSetupError =>
+      // if loader role is not granted to SYSADMIN, snowflake throws the below
+      // Table 'EVENTS' already exists, but current role has no privileges on it.
+      "Loader role has no privileges on events table. Role must be granted to SYSADMIN."
```

Comment:
```suggestion
      s"Loader role has no privileges on $tableName table. Role must be granted to SYSADMIN."
```
It'd be nice to extract the table name in the regex and put it in the message.
You could use `unapply` for that, like [here](https://github.com/snowplow-incubator/snowplow-lake-loader/blob/develop/modules/azure/src/main/scala/com.snowplowanalytics.snowplow.lakes/AzureApp.scala#L44-L47).